staging/src/k8s.io/client-go/transport/token_source.go

```diff
@@ -44,8 +44,8 @@ func TokenSourceWrapTransport(ts oauth2.TokenSource) func(http.RoundTripper) htt
 
 // NewCachedFileTokenSource returns a oauth2.TokenSource reads a token from a
 // file at a specified path and periodically reloads it.
-func NewCachedFileTokenSource(path string) oauth2.TokenSource {
-	return &cachingTokenSource{
+func NewCachedFileTokenSource(initialToken, path string) oauth2.TokenSource {
```

Comment:
Are you using this? I like the old signature a lot better.

Reply:
yes, missed the latest push. required to support a rest.Config that provides both a bearer token and bearer token file